WordPress/Classes/ViewRelated/Reader/DetailWebView/ReaderDetailHeaderView.swift

```diff
@@ -0,0 +1,326 @@
+                let end = gradientView.endPoint
+
+                gradientView.startPoint = end
+                gradientView.endPoint = start
+            }
+        }
+    }
+
+    private func configureTag() {
+        var tag = ""
+        if let rawTag = post?.primaryTag {
+            if rawTag.count > 0 {
+                tag = "#\(rawTag)"
+            }
+        }
+        tagButton.isHidden = tag.count == 0
+        tagButton.setTitle(tag, for: UIControl.State())
+        tagButton.setTitle(tag, for: .highlighted)
+    }
+
+    private func reloadGradientColors() {
+        bylineGradientViews.forEach({ view in
+            view.fromColor = .basicBackground
+            view.toColor = UIColor.basicBackground.withAlphaComponent(0.0)
+        })
+    }
+
+    private func prepareForVoiceOver() {
+        guard let post = post else {
+            blogNameButton.isAccessibilityElement = false
+            return
+        }
+
+        blogNameButton.isAccessibilityElement = true
+        blogNameButton.accessibilityTraits = [.staticText, .button]
+        blogNameButton.accessibilityHint = NSLocalizedString("Shows the site's posts.", comment: "Accessibility hint for the site name and URL button on Reader's Post Details.")
+        if let label = blogNameLabel(post) {
+            blogNameButton.accessibilityLabel = label
+        }
+    }
+
+    private func prepareMenuForVoiceOver() {
+        menuButton.accessibilityLabel = NSLocalizedString("More", comment: "Accessibility label for the More button on Reader's post details")
+        menuButton.accessibilityTraits = UIAccessibilityTraits.button
+        menuButton.accessibilityHint = NSLocalizedString("Shows more options.", comment: "Accessibility hint for the More button on Reader's post details")
+    }
+
+    private func blogNameLabel(_ post: ReaderPost) -> String? {
+        guard let postedIn = post.blogNameForDisplay(),
+            let postedBy = post.authorDisplayName,
+            let postedAtURL = post.siteURLForDisplay()?.components(separatedBy: "//").last else {
+                return nil
+        }
+
+        guard let postedOn = post.dateCreated?.mediumString() else {
+            let format = NSLocalizedString("Posted in %@, at %@, by %@.", comment: "Accessibility label for the blog name in the Reader's post details, without date. Placeholders are blog title, blog URL, author name")
+            return String(format: format, postedIn, postedAtURL, postedBy)
+        }
+
+        let format = NSLocalizedString("Posted in %@, at %@, by %@, %@", comment: "Accessibility label for the blog name in the Reader's post details. Placeholders are blog title, blog URL, author name, published date")
+        return String(format: format, postedIn, postedAtURL, postedBy, postedOn)
+    }
+
+    private func preparePostTitleForVoiceOver() {
+        guard let post = post else {
+            return
+        }
+
+        guard let title = post.titleForDisplay() else {
```

Comment:
This guard can be combined with the previous guard

Reply:
Fixed in 94b4b274bfc3f441327b4ef1a38c478852a4eea2